docs/site/components/language-switcher/language-switcher.html

```diff
@@ -74,25 +74,25 @@ <h4 class="list-item--heading">External Resources</h4>
       <h4 class="list-item--heading">gcloud Libraries</h4>
     </li>
     <li>
-      <a href="http://googlecloudplatform.github.io/gcloud-java" title="gcloud-java Documentation" class="skip-external-link">
+      <a href="http://googlecloudplatform.github.io/gcloud-java" target="_blank" title="gcloud-java Documentation" class="skip-external-link">
```

Comment:
These links are supposed to behave more like a dropdown than anchor tags.. so I don't think we want new window here.